models/staging/ecommerce/stg_ecommerce__products.sql

```diff
@@ -0,0 +1,20 @@
+
+select
+    -- primary key
+
+    -- foreign keys
+    product_id,
```

Comment:
this is your primary key!

Reply:
Updated in  [1ef6a62](https://github.com/mtla-saurabh/ae-capstone-sshahdeo/pull/2/commits/1ef6a62b1c8e5db2f1345638a99903f13121760e)